voicevox_engine/tts_pipeline/tts_engine.py

```diff
@@ -559,9 +560,14 @@ def update_length_and_pitch(
         accent_phrases = self.update_pitch(accent_phrases, style_id)
         return accent_phrases
 
-    def create_accent_phrases(self, text: str, style_id: StyleId) -> list[AccentPhrase]:
+    def create_accent_phrases(
+        self, text: str, style_id: StyleId, enable_e2k: bool = False
```

Comment:
ここも初期値なしが良いかも？
```suggestion
        self, text: str, style_id: StyleId, enable_e2k: bool = False  # TODO: 初期値をなくす？
```
（列数が長くなってるのでフォーマットが必要かもです 🙇 ）

Reply:
https://github.com/VOICEVOX/voicevox_engine/pull/1548/commits/c15073eb2f80309399992b187c3458402af8182b にて対応しました。